src/XMakeTasks/UnitTests/DirectoryBuildProjectImportTestBase.cs

```diff
@@ -20,7 +20,7 @@ abstract public class DirectoryBuildProjectImportTestBase : IDisposable
                     </PropertyGroup>
                 </Project>";
 
-        private const string ProjectRelativePath = @"src\foo\foo.csproj";
+        private readonly string _projectRelativePath = Path.Combine("src", "foo", "foo.csproj");
```

Comment:
Is the field rename required? It doesn't match the style being used for private fields in this file.